.devcontainer/devcontainer.json

```diff
@@ -1,13 +1,16 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
 // https://github.com/devcontainers/images/tree/main/src/javascript-node
 {
-  "name": "Node.js",
-  "build": {
-    "dockerfile": "Dockerfile",
-    // Update 'VARIANT' to pick a Node version: 18, 16, 14.
-    // Append -bullseye or -buster to pin to an OS version.
-    // Use -bullseye variants on local arm64/Apple Silicon.
-    "args": { "VARIANT": "16-bullseye" }
+  "name": "JavaScript Portfolio Site with GitHub Codespaces",
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
```

Comment:
it doesn't need to be the javascript-node image anymore?

Reply:
I took a queue from the Codespace Node based templates and are set to use this image. When I dug in, the image supports Node/JS/TypeScript (https://mcr.microsoft.com/en-us/product/devcontainers/universal/about). Tested in my fork and all worked great, so I made the switch.